wiki/Glossary/en.md

```diff
@@ -639,12 +639,6 @@ The total duration of a beatmap. This includes the lead-in time and the break ti
 
 The main ranking metric of the game.
 
-### puush
```

Comment:
I think you can keep the term in the glossary. Maybe add a note that it no longer accepts new users and is no longer in active development.

Reply:
agreed. please don't erase for existence :)